templates/markdown_editor/markdown_editor.html

```diff
@@ -0,0 +1,131 @@
+      height: 725px;
+      overflow-y: scroll;
+    }
+  </style>
+{% endblock %}
+
+{% block body %}
+  <script>
+    $(document).ready(function(){
+      $("#wmd-input-id_body").on("keyup", function() {
+        const csrfToken = "{{ csrf_token }}";
+        $.ajax({
+          url: "{{url('blog_preview')}}",
+          type: 'POST',
+          headers: {
+            'X-CSRFToken': csrfToken,  // Include the CSRF token in the headers
+          },
+          data: {
+            preview: $(this).val()
+          },
+          success: function(data) {
+            $('#display').html(data)
+          },
+          error: function(error) {
+            alert(error);
+            console.log(error.message)
+          }
+        })
+      });
+    });
+  </script>
+
+
+  <script>
+    document.addEventListener("DOMContentLoaded", function() {
+      const leftDiv = document.getElementById("wmd-input-id_body");
+      const rightDiv = document.getElementById("display");
+
+      leftDiv.addEventListener("scroll", function() {
+        rightDiv.scrollTop = leftDiv.scrollTop;
+      });
+
+      rightDiv.addEventListener("scroll", function() {
+        leftDiv.scrollTop = rightDiv.scrollTop;
+      });
+    });
+  </script>
+
+  <script src="{{ static('pagedown/Markdown.Converter.js') }}"></script>
+  <script src="{{ static('pagedown-extra/pagedown/Markdown.Converter.js') }}"></script>
+  <script src="{{ static('pagedown/Markdown.Sanitizer.js') }}"></script>
+  <script src="{{ static('pagedown/Markdown.Editor.js') }}"></script>
+  <script src="{{ static('pagedown-extra/Markdown.Extra.js') }}"></script>
+  <script src="{{ static('pagedown_init.js') }}"></script>
+  <script src="{{ static('mathjax3_config.js') }}"></script>
+  <script src="http://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"></script>
+  <script src="{{ static('pagedown_math.js') }}"></script>
+
+  <div id="new-comment" class="form-area" style="">
+    <form class="comment-submit-form" action="" method="post">
+      <input type="hidden" name="csrfmiddlewaretoken" value="Ngk7TfY2iIW4e9cPitl05k1c7xNUsVk9kCTkrAFveVmmcIhjRKv4GLtItKANAvae">
+      <input type="hidden" name="parent" id="id_parent">
+      <div class="comment-post-wrapper">
+        <div id="comment-form-body"><div class="wmd-wrapper image-upload-enabled">
+          <div class="wmd-panel">
+            <div id="wmd-button-bar-id_body"></div>
+            <textarea id="wmd-input-id_body" class="wmd-input" name="body" required=""></textarea>
+          </div>
+          <div id="id_body-preview" data-preview-url="/widgets/preview/comment" data-textarea-id="wmd-input-id_body" data-timeout="1000" class="wmd-panel wmd-preview dmmd-preview dmmd-no-button">
+            <div class="dmmd-preview-update"><i class="fa fa-refresh"></i> Update Preview</div>
+            <div class="dmmd-preview-content content-description"></div>
+          </div>
+          <div class="pagedown-image-upload">
+            <h2>Insert Image</h2>
```

Comment:
mấy string tiếng Anh này em viết thành: `{{_('Insert Image')}}` để dịch sang tiếng Việt.
Để dịch sang tiếng Việt thì em làm các thao tác sau (copy từ readme)

Update vietnamese translation:
- python3 manage.py makemessages
- go to locale/vi
- modify .po file
- python3 manage.py compilemessages
- python3 manage.py compilejsi18n